templates/launch.html

```diff
@@ -160,21 +172,71 @@ <h3>Runner Data</h3>
         document.querySelector('#survey_metadata_fields').innerHTML = ""
     }
 
+    function showAllDivs() {
+        document.querySelector("#launch_pattern").setAttribute("disabled", "disabled")
+
+        document.querySelectorAll("#survey_metadata, .hidden-field-container, .group-title", ).forEach(function (element) {
+            element.style.display = 'block';
+        });
+    }
+
     function includeSurveyMetadataFields(schema_name, survey_type) {
+        let launchPattern = document.querySelector("#launch_pattern").value
         let eqIdValue = schema_name.split('_')[0]
         let formTypeValue = schema_name.split("_").slice(1).join("_")
 
-        document.querySelector('#survey_metadata_fields').innerHTML = `
-            <h3>${survey_type} Survey Metadata</h3>
-            <div class="field-container">
-                <label for="eq_id">eq_id</label>
-                <input id="eq_id" name="eq_id" type="text" value="${eqIdValue}" class="qa-eq_id">
-            </div>
-            <div class="field-container">
-                <label for="form_type">form_type</label>
-                <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
-            </div>
-        `
+        document.querySelector('#survey_metadata_fields').style.display = "block";
+
+        if (launchPattern === "v1") {
+            document.querySelector('#survey_metadata_fields').innerHTML = `
+                <h3>${survey_type} Survey Metadata</h3>
+                <div class="field-container" class="visible-block">
+                    <label for="eq_id">eq_id</label>
+                    <input id="eq_id" name="eq_id" type="text" value="${eqIdValue}" class="qa-eq_id" >
+                </div>
+                <div class="field-container" class="visible-block">
+                    <label for="form_type">form_type</label>
+                    <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
+                </div>
+            `
+        } else {
+            document.querySelector('#survey_metadata_fields').innerHTML = `
+                <h3>${survey_type} Survey Metadata</h3>
+                <div class="field-container" class="visible-block">
+                    <label for="form_type">form_type</label>
+                    <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
+                </div>
+            `
+        }
+
+    }
+
+    function clearSchemaNameSelection() {
```

Comment:
Is this function needed anymore if we are making asking you to select the launch pattern upfront, which you can't change without a page reload anyway?.

Reply:
Perhaps need to update the name of this, as some of this is still required, e.g. this is currently stopping social schemas from being laucnhed in v1 mode for example